sdk/storage/AzureStorageBlob/Source/StorageStreamUploader.swift

```diff
@@ -274,9 +274,6 @@ internal class BlobStreamUploader: BlobUploader {
         options: UploadBlobOptions? = nil
     ) throws {
         let manager = FileManager.default
-        guard manager.fileExists(atPath: source.path) else {
-            throw AzureError.fileSystem("File not found: \(source.path)")
-        }
```

Comment:
Remove the code to swallow this error, as it masked the real problem (permissions).